Backend/app/executors/timer.py

```
@@ -0,0 +1,79 @@
+
+            if target <= now:
+                from datetime import timedelta
+                target = target + timedelta(days=1)
+
+            wait_seconds = (target - now).total_seconds()
+
+            if wait_seconds > 86400:
+                raise HTTPException(
+                    status_code=400,
+                    detail="Wait time exceeds 24 hours"
+                )
```

Ruff (0.14.10)

Abstract `raise` to an inner function

(TRY301)